test/unit/org/apache/cassandra/service/QueryInfoTrackerTest.java

```diff
@@ -112,6 +112,56 @@ public void testSimpleQueryTracing()
         assertEquals(0, tracker.loggedWrites.get());
     }
 
+    @Test
+    public void testReadQueryTracingWithStaticRows()
+    {
+        int keys = 4;
+        int clustering = 3;
+        String table = KEYSPACE + ".qit_read_static";
+        session.execute("CREATE TABLE " + table + "(k int, c int, v int, sv int static, PRIMARY KEY (k, c))");
+        for (int k = 0; k < keys; k++)
+        {
+            for (int c = 0; c < clustering; c++)
+            {
+                session.execute("INSERT INTO " + table + "(k, c, v, sv) values (?, ?, ?, ?)", k, c, k, k * 77);
+            }
+        }
+
+        assertEquals(0, tracker.reads.get());
+        session.execute("SELECT * FROM " + table + " WHERE k = ?", 0);
+        assertEquals(1, tracker.reads.get());
```

Comment:
So without the new `applyToStatic` override, this was 0 still?

Reply:
No, it was 1. `tracker.reads` here means the number of single partition reads executed. It is bumped in `onDone`.

Without `applyToStatic` the following condition is failing:
`assertEquals(1 + clustering, tracker.readRows.get());`
I.e. only `clustering` rows are read (the static one is not accounted for)

Reply:
oh.  Then I think we were already doing the right thing.  We should not be adding extra rows for the fact that something is static.  The number of rows counted should equal the number of rows in the result set for this.

Reply:
whoa... Does it mean there's a bug in DSE?
https://github.com/riptano/bdp/blob/10c45dae42b05a95d5aca1f5d52e009bc60618e5/dse-db/src/java/org/apache/cassandra/service/ResponseResolver.java#L100-L107

Let's get back to this discussion tomorrow.

Reply:
We agreed to make C* work like DSE, and to add static columns to the list of things we need to think about how we bill for again